flake/dev/new-plugin.py

```diff
@@ -27,10 +26,32 @@
   empty = {{
     plugins.{name}.enable = true;
   }};
+
+  default = {{
+    plugins.{name} = {{
+      enable = true;
+      settings = {{
+        exampleOption = false;
+      }};
+    }};
+  }};
 }}
 """
 
 
+def normalized_name(name) -> str:
+    """
+    Convert name to a normalized package name format.
+
+    Args:
+        name (str): The original name to normalize.
+
+    Returns:
+        str: The normalized package name.
+    """
+    return name.replace(".", "-")
```

Comment:
The kebab_case function _claims_ it will "replace non-alphanumeric chars with hyphens".

If that's accurate, is this redundant?

If that's inaccurate, maybe it should be fixed there instead? Or is there a reason I'm missing to do this in a separate function?

Reply:
It does more than that, though. It's our function for stripping `nvim` We don't do that upstream in nixpkgs. Maybe we can split that into 2 functions.